requirements/edx-sandbox/base.txt

```diff
@@ -1,8 +1,7 @@
 #
-# This file is autogenerated by pip-compile
-# To update, run:
-#
-#    make upgrade
+# This file has been frozen for the benefit of Python 2.7 sandboxes, and
+# "make upgrade" no longer updates it.  It can be removed once that
+# version of the sandbox is no longer run in production.
```

Comment:
When we remove this, we'll want to update configuration to use the new py35 version of the sandbox requirements as well: https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L1628

Can you add that to the comment and also add the note that, the cutover should be announced and communicated to partners in advance of making the actual change to use a new requirements set in sandboxes.